frontend/src/components/Nav.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import '../App.global.scss';
+import { Link } from 'react-router-dom';
+import { Avatar } from 'wix-style-react';
+
+function Nav() {
```

Comment:
I think we need to decide whether we use arrow functions or standard functions for our components. Arrow functions seems cleaner and more modern to me